databricks_cli/unity_catalog/perms_cli.py

```diff
@@ -67,19 +67,23 @@ def _get_perm_securable_name_and_type(catalog_name, schema_full_name, table_full
 @click.option('--external-location', cls=OneOfOption, default=None,
               one_of=PERMISSIONS_OBJ_TYPES,
               help='Name of the external location of interest')
+@click.option('--effective', '-e', is_flag=True, default=False)
```

Comment:
The short form option doesn't seem like a good idea, since it naturally conflicts with `--external-location`.

WDYT?

Reply:
Good point. I removed the `-e` flag (and added some help text) for that command.